my_game.py

```diff
@@ -385,6 +415,11 @@ def on_show_view(self):
         # to reset the viewport back to the start so we can see what we draw.
         arcade.set_viewport(0, self.window.width, 0, self.window.height)
 
+        self.player_amount = 1
+        self.player_sprite_list = arcade.SpriteList()
+
+        self.player_sprite_list = create_players(self.player_amount)
```

Comment:
Huh? I thought the number of players could be selected. Here, it's hard coded to 1??